spec/template.dd

```diff
@@ -777,30 +784,33 @@ $(GNAME TemplateSequenceParameter):
     $(P An $(I AliasSeq) can be used as an argument list to instantiate
         another template, or as the list of parameters for a function.)
 
+        $(SPEC_RUNNABLE_EXAMPLE_RUN
         ---
+        import std.stdio;
+
         template print(args...)
         {
-            void print()
+            void f()
             {
                 writeln("args are ", args); // args is a ValueSeq
             }
         }
 
-        template write(Args...)
+        template write(Args...) // Args is a TypeSeq
         {
-            void write(Args args) // Args is a TypeSeq
-                                  // args is a ValueSeq
+            void f(Args args) // args is a ValueSeq
             {
                 writeln("args are ", args);
             }
         }
 
         void main()
         {
-            print!(1,'a',6.8).print();                    // prints: args are 1a6.8
-            write!(int, char, double).write(1, 'a', 6.8); // prints: args are 1a6.8
```

Comment:
This is interesting, an eponymous function template cannot be called explicitly, it tries to call `write!(int, char, double)().write(1, 'a', 6.8);` and says it's missing arguments. I wonder if that's a bug.

Reply:
I think it's because of implicit parentheses for function calls.